package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "auth0-source-control-extension-tools",
-  "version": "5.0.0",
+  "version": "5.1.0",
```

Comment:
For maintainers: It looks like 5.0.0 is only available on a beta branch of deploy-cli? (ref. https://github.com/auth0/auth0-deploy-cli/pull/306). This change needs to go out before actions beta ends, can you confirm if I should point this change to the `v4.x` branch of this repo and minor-bump the version on 4.x.

Reply:
@pmalouin i'm checking on this with the engineers who worked on the beta of deploy-cli to see if we can pull in v5.x of this lib without breaking anything. If not, we might have to also maintain a beta branch in this repo.

Reply:
@pmalouin this should be fine, i'll write a comment on the PR with some more information.